client/blocks/eligibility-warnings/hold-list.jsx

```diff
@@ -32,7 +32,7 @@ function getHoldMessages( siteSlug, translate ) {
 		},
 		SITE_PRIVATE: {
 			title: translate( 'Private site not supported' ),
-			description: translate( 'Make your site public to resolve.' ),
+			description: translate( 'Make your site public or hidden to resolve.' ),
```

Comment:
Drive-by comment: I think this string isn't clear enough. "To resolve"? What does that mean exactly here?